salesforce.ApexPage Exception {
  apiVersion = 37
  availableInTouch = false
  confirmationTokenRequired = false
  description = "Default Force.com page for post-authentication errors"
  label = "Exception"
  packageVersions = [
    {
      majorNumber = 3
      minorNumber = 4
      namespace = salesforce.InstalledPackage.instance.npe01
    },
    {
      majorNumber = 3
      minorNumber = 5
      namespace = salesforce.InstalledPackage.instance.npe03
    },
    {
      majorNumber = 3
      minorNumber = 3
      namespace = salesforce.InstalledPackage.instance.npe4
    },
    {
      majorNumber = 3
      minorNumber = 3
      namespace = salesforce.InstalledPackage.instance.npe5
    },
    {
      majorNumber = 3
      minorNumber = 6
      namespace = salesforce.InstalledPackage.instance.npo02
    },
    {
      majorNumber = 3
      minorNumber = 82
      namespace = salesforce.InstalledPackage.instance.npsp
    },
    {
      majorNumber = 1
      minorNumber = 11
      namespace = salesforce.InstalledPackage.instance.sf_chttr_apps
    },
    {
      majorNumber = 5
      minorNumber = 201
      namespace = salesforce.InstalledPackage.instance.stayclassy
    },
  ]
  fullName = "Exception"
  content = file("salesforce/Records/ApexPage/Exception.page")
  _generated_dependencies = [
    {
      reference = salesforce.ApexComponent.instance.SitePoweredBy
    },
    {
      reference = salesforce.StaticResource.instance.SiteSamples
    },
  ]
  _alias = "Exception"
}
